src/profile-logic/import/instruments/MaybeCompressedReader.js

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+import * as pako from 'pako';
```

Comment:
We should probably prioritize splitting out this dependency so we don't affect our bundle size even more. We still have an outstanding issue related to another zip library we use. I think this is the third decompression library we are using (which is fine). However we should ensure that we're not negatively affecting the load speed from adding more.

See issue #817.

/cc @julienw

Reply:
Sure. Thanks @gregtatum . I will check this out.

Reply:
Actually we already include `pako` because `jszip` depends on it ;)
Maybe make sure we use the same version of it? (`yarn.lock` should tell you that).

Also (not for this PR) maybe we could also use `pako` instead of our own compiled version of zlib in the worker.

Reply:
I don't see a `yarn.lock` change despite the dependency change in `package.json`, have you forgotten to add the file change to the PR?

Reply:
@julienw, I didn't actually know that `yarn.lock` file should also be included in a commit.
Will make a commit for this.

Reply:
@julienw I changed `pako` version to 1.0.2 (`Jszip` internally uses `pako`'s same version)
So now `yarn.lock` the file is unchanged.

Reply:
As a quick note, it's fine if `yarn.lock` is changed as long as all `pako` dependencies link to the same version.

Here is our current entry for pako in `yarn.lock`:
```
pako@~1.0.2, pako@~1.0.5:
  version "1.0.6"
  resolved "https://registry.yarnpkg.com/pako/-/pako-1.0.6.tgz#0101211baa70c4bca4a0f63f2206e97b7dfaf258"
  integrity sha512-lQe48YPsMJAig+yngZ87Lus+NF+3mtu7DVOBu6b/gHO1YpKwIj5AWjZ/TOS7i46HD/UixzWb1zeWDZfGZ3iYcg==
```

This means there are 2 different dependency versions: `~1.0.2` and `~1.0.5`. Yarn managed to resolve these 2 versions to the same version `1.0.6`.
It all depends on the dependencies: can they all be resolved to the same version?
Also we can look closer at what exactly requests these versions. `yarn why pako` gives this information:
```
$ yarn why pako
yarn why v1.15.2
[1/4] Why do we have the module "pako"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "pako@1.0.6"
info Reasons this module exists
   - "jszip" depends on it
   - Hoisted from "jszip#pako"
   - Hoisted from "webpack#node-libs-browser#browserify-zlib#pako"
info Disk size without dependencies: "840KB"
info Disk size with unique dependencies: "840KB"
info Disk size with transitive dependencies: "840KB"
info Number of shared dependencies: 0
```

Here we see that our direct dependencies jszip and webpack transitively depends on `pako`. We also know that `webpack` won't be in our build result, so actually it could resolve to another `pako` version without impacting our build size.

So now you have the big picture :)
1.0.2 sounds good but will likely resolve to 1.0.6. If you chose the latest 1.0.10 then probably everything will resolve to 1.0.10, as it is semver-compatible with the other dependencies. This sounds good too as this will make us have the latest and greatest version :-)

Hope this helps!

Reply:
Wow!!! Thanks for the detailed explanation @julienw ! :)
I will update the version to 1.0.10 and push the changes accordingly.